Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml

```diff
@@ -1555,25 +1503,25 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  '161':
-    id: '161'
-    taskid: 99ac1978-a22e-435b-8618-5665028eb567
+  "161":
+    id: "161"
+    taskid: f7b5c63c-7cbb-4907-8317-2dd39f6ed397
     type: regular
     task:
-      id: 99ac1978-a22e-435b-8618-5665028eb567
+      id: f7b5c63c-7cbb-4907-8317-2dd39f6ed397
       version: -1
       name: Set detected usernames
       description: Sets a value into the context with the given context key
       scriptName: SetAndHandleEmpty
       type: regular
       iscommand: false
-      brand: ''
+      brand: ""
     nexttasks:
       '#none#':
-      - '162'
+      - "163"
```

Comment:
162 is get hosrname and 163 is to set the hostnames. 

I think we should change the order (162 first).